tutorials/xr/setting_up_xr.rst

```diff
@@ -40,7 +40,8 @@ While in Godot 3 most things worked out of the box, Godot 4 needs a little more
 
 .. note::
     Godot 4 has 3 renderer options, Compatibility, Mobile, and Forward+. In the future XR desktop projects should use Forward+, and projects for stand-alone headsets
-    should use Mobile. However Compatibility is the recommended renderer for now due to it having the best XR performance.
+    should use Mobile. Currently Compatibility is the recommended renderer for stand-alone headsets, and ironically Mobile is the recommended renderer for desktop.
```

Comment:
```suggestion
    should use Mobile. Currently Compatibility is the recommended renderer for standalone headsets, and ironically Mobile is the recommended renderer for desktop.
```
Single word, not hyphenated.

```suggestion
    should use Mobile. Currently Compatibility is the recommended renderer for standalone headsets, and Mobile is the recommended renderer for desktop XR. Forward+ *can* run XR, but struggles with performance.
```
Optional suggestion. Subjectively I'm not a fan of "ironically" here.

Reply:
Normally I wouldn't want to be casual and use "ironically". However I feel like if we don't put in something to say "we didn't mix up the suggestions" people are gonna assume we accidentally swapped the recommendations, because you would logically assume Mobile is for stand-alone headsets.

Reply:
Makes sense, like I said, subjective and optional suggestion